.github/actions/spelling/patterns.txt

```diff
@@ -19,3 +19,6 @@
 [Xx][Uu]nit
 [Xx][Pp][Aa][Tt][Hh]
 [Rr]eadme
+[{]?[0-9a-zA-Z]{8}-([0-9a-zA-Z]{4}-){3}[0-9a-zA-Z]{12}[}]?
+[s][h][a][0-9]{3}-[0-9a-zA-Z\+\/]{64}
+\b([a-f0-9]{40})\b
```

Comment:
Regular expressions for the following cases:
1. GUIDs
2. CDN Integrity "sha" IDs, which do have a proper name that I forgot. Found in ResourceManagementOptions classes. For example:
![image](https://user-images.githubusercontent.com/51411356/204086057-5492f4b5-cf60-4d59-ae78-c9978614335b.png)
3. Git commit hashes.

Reply:
Very nice!